x-pack/plugins/timelines/public/components/hover_actions/actions/add_to_timeline.tsx

```diff
@@ -46,6 +47,17 @@ const useGetHandleStartDragToTimeline = ({
   return handleStartDragToTimeline;
 };
 
+const ToastAddSuccess: React.FC<{
```

Comment:
Consider renaming this to something like `AddSuccessMessage` as this component is not handling anything related to a toast. 

You can also consider passing `children instead of props` so that any message can be passed as below
```jsx
<ToastAddSuccess>
{i18n.ADDED_TO_TIMELINE_OR_TEMPLATE_MESSAGE(
    addSuccess,
    timelineType === 'default'
  )}
</ToastAddSuccess>
```


`nitpick`: Another suggestion is to create a generic Component `BreakWordText` which can receive message as children and can be used anywhere where we need to handle long text.